### PR TITLE
HPCC4J-321 Introducing hpcc4j

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: Rodrigo Pastrana <Rodrigo.Pastrana@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by rpastrana** [comment]
@g-pan please re-review the doc changes. Thanks.

**Comment by rpastrana** [comment]
@Michael-Gardner please test/review

File: README.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
-  - [Sample code](https://github.com/hpcc-systems/HPCC-JAPIs/blob/master/wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java)
+  - [Sample code](https://github.com/hpcc-systems/hpcc4j/blob/master/wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java)
 
 - DFS Client (./dfdsclient)
-  - Java based interface into HPCC's dfs backend [README](https://github.com/hpcc-systems/HPCC-JAPIs/tree/master/dfsclient/README.md)
+  - Java based interface into HPCC's dfs backend [README](https://github.com/hpcc-systems/hpcc4j/tree/master/dfsclient/README.md)
 
 - HPCC Commons (./commons-hpcc)
-  - HPCC Systems Java API common classes [README](https://github.com/hpcc-systems/HPCC-JAPIs/blob/master/commons-hpcc/README.md)
+  - HPCC Systems Java API common classes [README](https://github.com/hpcc-systems/hpccj4/blob/master/commons-hpcc/README.md)
```
Comment (g-pan) [inline README.md:26]: should this be hpcc4j?

**Review by g-pan — COMMENTED** [review]
Only one issue - see comment in the README.md file

File: README.md (hunk 1 above)
Comment (rpastrana) [reply README.md:26]: yep, good catch

**Comment by Michael-Gardner** [comment]
@rpastrana seems to function fine on my box.  Tested again last night.  Gave the source code changes a read and it looks good also.

**Pull request merged**